ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,9 +32,8 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
```

Comment:
You can run this test locally and fix it, removing this ignore should now be breaking it when you try running it.

Reply:
Taken time to understand this after running the test locally without @ignore! Ideally this test in the EndVistTest was already written But lacks  its definition that why its annotated with @ignore so that it is skipped when running the folder as Junit test. So am required to write its definition, right? This suggests to me that nothing to change in this test except refactoring out the annotation that ignores it after meeting its definition. @k-joseph @HerbertYiga

Reply:
when a test is ignored, the compiler skips it and runs others not ignored, it's ignored because it should fail when not ignored, work towards fixing it such that it passes when not ignored.

Reply:
I must also say that keep up the openness, communicating whenever you are blocked or not sure what to do will get you exposed to a lot to learn quickly

Reply:
> when a test is ignored, the compiler skips it and runs others not ignored, it's ignored because it should fail when not ignored, work towards fixing it such that it passes when not ignored.

I get it now! Thanks @k-joseph

Reply:
> I must also say that keep up the openness, communicating whenever you are blocked or not sure what to do will get you exposed to a lot to learn quickly

Welcome!